lib/Conch/Route.pm

```diff
@@ -71,6 +71,9 @@ sub all_routes (
     $root->post('/logout')->to('login#session_logout');
     $root->post('/reset_password')->to('login#reset_password');
 
+    # GET /schema/:input_or_response/:schema_name
+    $root->get('/schema/:input_or_response/:name')->to('schema#get');
```

Comment:
You may want to put the `(request|response)` restriction right here so the controller doesn't need to check it:  https://metacpan.org/pod/Mojolicious::Guides::Routing#Restrictive-placeholders

Reply:
Fixed in fdeef31